### PR TITLE
Added new content folder for static template images

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

**Review by jjeroch — CHANGES_REQUESTED** [review]
a couple of images are still to be added (the once which are located in the page "company roles"

**Comment by jjeroch** [comment]
@manojava-gk  do we have a related pr on frontend side to ensure that images are correctly fetched?

**Comment by manojava-gk** [comment]
> @manojava-gk do we have a related pr on frontend side to ensure that images are correctly fetched?

Front end side yes. here is the link - https://github.com/catenax-ng/tx-portal-frontend/pull/228

**Review by oyo — APPROVED** [review]
I don't approve that `imageUrl` path prefix. Prefer convention over configuration. Also a url path starting with `/images/content/` doesn't exist

File: public/assets/content/de/companyroles.json [hunk 1]
```diff
@@ -357,22 +357,22 @@
       {
         "title": "Marketplace Subscriptions",
         "description": "Inside the app and service marketplace, subscriptions to avialable offers can get made. Find out more how this works by following the details below",
-        "imageUrl": "teaser.png",
+        "imageUrl": "/images/content//images/content/teaser.png",
```
[the remark this answers is not in the document]
Comment (manojava-gk) [reply public/assets/content/de/companyroles.json:360]: Initially I was expecting http url. For that reason it is named as imageUrl. But it is no more http url. It is just a path. I can update the attribute name here

**Comment by manojava-gk** [comment]
Shall I merge this PR? @oyo

**Review by oyo — APPROVED** [review]

**Pull request merged**